### ISSUE TITLE
Event Details - Responses

### ISSUE THREAD

**Issue by chiwaileung** [MSFREACH/MSF-REACH-COLLABORATION#145]
- [x] Alignment on tabbed table and map is wrong
- [x] Tab Header title should be all capital letters
- [x] Height of Tab header is different
- [x] Edit button should be rounded
- [x] Table content format is wrong, refer to sketch
- [x] Sharepoint link should be a icon

- UAT
![image](https://user-images.githubusercontent.com/50646734/72783989-b4713100-3c62-11ea-9431-9b1f9e740cae.png)

-Sketch
![image](https://user-images.githubusercontent.com/50646734/72784007-bfc45c80-3c62-11ea-8390-b946b14e3f47.png)

**Comment by hohangleung** [comment]
Table content ordering and grouping are based on "MSF_EventDetails_Structure_Content.xlsx", therefore there are difference from the sketch

**Comment by chiwaileung** [comment]
> Table content ordering and grouping are based on "MSF_EventDetails_Structure_Content.xlsx", therefore there are difference from the sketch

@LucieGueuning Please confirm.

**Comment by LucieGueuning** [comment]
> > Table content ordering and grouping are based on "MSF_EventDetails_Structure_Content.xlsx", therefore there are difference from the sketch
> 
> @LucieGueuning Please confirm.

Yes, Table content Ordering and grouping are based on the .xlsx file. BUT Where are you taking the content for dropdown and categories names, @hohangleung ?
